clang/lib/Format/UnwrappedLineFormatter.cpp

```diff
@@ -1355,6 +1355,8 @@ unsigned UnwrappedLineFormatter::format(
     bool FixIndentation = (FixBadIndentation || ContinueFormatting) &&
                           Indent != TheLine.First->OriginalColumn;
     bool ShouldFormat = TheLine.Affected || FixIndentation;
+    if (Style.IgnorePPDefinitions && TheLine.Type == LT_PreprocessorDirective)
```

Comment:
Wouldn't this take all preprocessor directives from the formatting? This is at least not what the documentation states.